### ISSUE TITLE
Cannot add multiple params

### ISSUE THREAD

**Issue by ehamberg** [emreyalvac/firebase-rs#18]
Shouldn't this `set_query`

https://github.com/emreyalvac/firebase-rs/blob/c1c32b6ada5ba72154c9d5aef53a380cf7e4e8e6/src/params.rs#L22-L23

…be something like the following?

```rust
    self.uri.query_pairs_mut().append_pair(k, v);
```

As it is now, if I add multiple parameters, like this:

```rust
.with_params()
.limit_to_first(50)
.start_at(1674526969)
.order_by("\"txid\"")
.finish()
```

Only the last one of them gets added to the URL.

**Comment by emreyalvac** [comment]
Hi,

Good catch. I will fix it.

**Comment by emreyalvac** [comment]
Hi @ehamberg,

Fixed in 2.0.6.

https://github.com/emreyalvac/firebase-rs/commit/f38e8c38d5b4c960ce6d4ced9546d30a4aaaffe6
